src/createServer.js

```diff
@@ -1,3 +1,39 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable max-len */
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { validateError } = require('./validateError');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-type', 'application/json');
+
+    const parsedUrl = new URL(req.url, 'http://localhost:5700');
+    const toCase = parsedUrl.searchParams.get('toCase');
+    const originalText = parsedUrl.pathname.slice(1);
+    const validateErrorMessage = validateError(originalText, toCase);
+
+    if (validateErrorMessage.errors.length === 0) {
+      const convertedCase = convertToCase(parsedUrl.pathname.slice(1), toCase);
+      const convertedText = convertedCase.convertedText;
+      const result = {
+        originalCase: convertedCase.originalCase,
+        targetCase: toCase,
+        originalText,
+        convertedText,
+      };
+
+      res.statusCode = 200;
+      res.statusMessage = 'Ok';
+      res.end(JSON.stringify(result));
+    }
+
+    if (validateErrorMessage.errors.length > 0) {
```

Comment:
Would be better to first check if some errors exist should throw an error, otherwise return the response with result